components/campaign/CampaignChaosBagView/index.tsx

```diff
@@ -207,19 +225,6 @@ class CampaignChaosBagView extends React.Component<Props, State> {
     }
   }
 
-  // _mutateCount = (id: ChaosTokenType, mutate: (count: number) => number) => {
```

Comment:
Can we take this out, or is this still being worked on?

Reply:
This was removed.